### PR TITLE
Task 1: Create a Maven project with basic testng and selenium setup

### PR DESCRIPTION
**Pull request by EmirSeho**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -12,6 +12,47 @@
...
+            <groupId>io.github.bonigarcia</groupId>
+            <artifactId>webdrivermanager</artifactId>
+            <version>${webdrivermanager.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.testng</groupId>
+            <artifactId>testng</artifactId>
+            <version>${testng.version}</version>
+            <scope>compile</scope>
```
Comment (nejlamaksumic-abh) [inline pom.xml:37]: why the `compile` scope?

File: src/test/java/com/emirseho/qamp/placelab/tests/SmokeTest.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+import com.emirseho.qamp.placelab.utils.WebDriverSetup;
+import org.openqa.selenium.WebDriver;
+import org.testng.annotations.AfterSuite;
+import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.Test;
+
+public class SmokeTest {
+    private WebDriver driver;
+
+    @BeforeSuite
```
Comment (nejlamaksumic-abh) [inline src/test/java/com/emirseho/qamp/placelab/tests/SmokeTest.java:12]: Why BeforeSuite and not BeforeTest?

**Review by nejlamaksumic-abh — COMMENTED** [review]
Overall good! Just please make sure to name your pull requests accordingly, for example this one could have been: `Task 1: Create a Maven project with basic testng and selenium setup` Same is for the branch name, could have been named `task1` Also, no need to commit the .idea files at all, those are just the files that are specific to your development environment, could have been added to the .gitignore

File: pom.xml (hunk 1 above)
Comment (EmirSeho) [reply pom.xml]: Not sure how that got there. I have removed line 37 completely.

**Review by nejlamaksumic-abh — APPROVED** [review]

File: src/test/java/com/emirseho/qamp/placelab/tests/SmokeTest.java (hunk 2 above)
Comment (EmirSeho) [reply src/test/java/com/emirseho/qamp/placelab/tests/SmokeTest.java]: This one was not intentional, changed to @BeforeTest.

**Pull request merged**